terragrunt/accounts/legacy/crates-io-prod/deployed-ref

```diff
@@ -0,0 +1 @@
+25e405013655cafcf1eb6c2ce2423914a0df79eb
```

Comment:
This is now referencing a commit in the same branch. I think we should update this to the merge commit once the PR has been merged into `master`.